script/seed.js

```diff
@@ -20,6 +52,16 @@ const seed = async () => {
         return Doki.create(doki);
```

Comment:
`const [ catDoki, rabbitDoki ]  = await Promise.all(dokis.map(doki => Doki.create(doki));`

It will also make the code more readable since you'll be able to easily tell where the doki seeds are created vs the user seeds